help/change-the-stream-description.md

```diff
@@ -2,9 +2,10 @@
 
 {!admin-only.md!}
 
-Streams descriptions are displayed when viewing the stream in the
-web and desktop apps, and can be used to explain the purpose of a
-stream and link to usage guidelines, resources, or related streams.
+Streams descriptions are displayed if you are in a stream view in the
+web or desktop app, and when viewing all streams or the stream settings
+menu in the mobile app. Streams descriptions can be used to explain the
```

Comment:
So ... stream descriptions are also shown in the web and desktop apps in the browse all streams panel and the streams settings panel/menu. Maybe we don't need to say where they are visible anymore since they are visible in many places? And instead we just say what they can be used for?

@alya mentioning you for your thoughts here.

> Stream descriptions can be used to explain the purpose of a stream and link to usage guidelines, resources, or related streams. Stream descriptions are shown when browsing all streams or viewing the stream's settings in the web, desktop, and mobile apps, and they are visible in the stream view of the web and desktop apps.

Reply:
Agreed -- let's just cut the sentence that lists where stream descriptions are shown.